### DISCUSSION TITLE
Welcome to Security Onion Discussions!

### DISCUSSION THREAD

**Discussion by dougburks** [Security-Onion-Solutions/securityonion#1283] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by TOoSmOotH** [comment]
Thanks @dougburks !!

**Reply by dougburks** [reply]
You're welcome @TOoSmOotH !

**Comment by weslambert** [comment]
Testing

On Wed, Sep 2, 2020 at 2:28 PM Doug Burks <notifications@github.com> wrote:

> You're welcome @TOoSmOotH <https://github.com/TOoSmOotH> !
>
> —
> You are receiving this because you are subscribed to this thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/Security-Onion-Solutions/securityonion/discussions/1283#discussioncomment-60643>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AEAM3KDMU3QTDRE3WI5CNT3SD2FGRANCNFSM4QTGBE7Q>
> .
>


-- 
https://twitter.com/therealwlambert
https://securityonion.net/

**Reply by dougburks** [reply]
It works!

**Reply by defensivedepth** [reply]
How does this look?

**Comment by defensivedepth** [comment]
@phil1090 What do you think?

**Reply by phil1090** [reply]
Thrilled to my core

**Comment by dlee35** [comment]
I believe Phil would concur

V/R,

Dustin

**Comment by skysbsb** [comment]
Hi!! Found that today at SOC =)

**Comment by darth-drew** [comment]
Way to go Security Onion Solutions!!

**Comment by jpancrazio** [comment]
Stupid question about SO 2.3 SO-Analyst workstation . Working on change some defaults of the VM .. just stupid little stuff but in settings , if i try to change clock and click unlock, it asks for Administrator authentication , but entering my current users password it doesn't seem to work . My current user can SUDO and the password works , also if i su administrator it says the user doesn't exist .. anyone have some insight ?

**Reply by jpancrazio** [reply]
Never mind .. i was able to su to root and reset the root password and that password allowed me to unlock the settings .. Thanks

**Reply by dougburks** [reply]
In the future, please create a new discussion rather than asking questions on this Welcome discussion.

Thanks!

**Comment by secraven** [comment]
Where can I find domain stats information in Kibana? The older version had its own dashboard.  I can't find a similar dashboard in the version.

https://docs.securityonion.net/en/2.3/domainstats.html?highlight=domain%20stats

**Reply by dougburks** [reply]
Please create a new discussion rather than asking questions on this Welcome discussion. Thanks!
